src/coreclr/inc/corcompile.h

```diff
@@ -149,17 +159,18 @@ enum CORCOMPILE_FIXUP_BLOB_KIND
     ENCODE_CHECK_VIRTUAL_FUNCTION_OVERRIDE,         /* Generate a runtime check to ensure that virtual function resolution has equivalent behavior at runtime as at compile time. If not equivalent, code will not be used */
     ENCODE_VERIFY_VIRTUAL_FUNCTION_OVERRIDE,        /* Generate a runtime check to ensure that virtual function resolution has equivalent behavior at runtime as at compile time. If not equivalent, generate runtime failure. */
 
+    ENCODE_CHECK_IL_BODY,                           /* Check to see if an IL method is defined the same at runtime as at compile time. A failed match will cause code not to be used. */
+    ENCODE_VERIFY_IL_BODY,                          /* Verify an IL body is defined the same at compile time and runtime. A failed match will cause a hard runtime failure. */
+
     ENCODE_MODULE_HANDLE                = 0x50,     /* Module token */
     ENCODE_STATIC_FIELD_ADDRESS,                    /* For accessing a static field */
-    ENCODE_MODULE_ID_FOR_STATICS,                   /* For accessing static fields */
     ENCODE_MODULE_ID_FOR_GENERIC_STATICS,           /* For accessing static fields */
```

Comment:
Don't we need to preserve the numeric value after removing `ENCODE_MODULE_ID_FOR_STATICS`?

Reply:
No, this is a dead encoding not used by anything anymore, and it never was used in R2R. (There are several here, which are legacies of the old NGEN code which I haven't gotten around to deleting. I deleted this one as I would have had to fix it otherwise, and that just isn't worth it.)

Reply:
Just to clarify, I am worried that the not dead encoding `ENCODE_MODULE_ID_FOR_GENERIC_STATICS` changed from 0x53 to 0x52. If we had a ready-to-run compiled binary encoded a `ENCODE_MODULE_ID_FOR_GENERIC_STATICS` fixup as 0x53 out there, it would be reinterpreted as `ENCODE_CLASS_ID_FOR_STATICS` in the new version of the runtime.

Reply:
All of the ENCODE_* values from ENCODE_MODULE_HANDLE onwards are no longer in use and were exclusive to NGEN.